java/ChartboostMediationDemoJava/src/main/java/com/example/chartboost/mediation/sdk/demo/java/AdController.java

```diff
@@ -0,0 +1,276 @@
+package com.example.chartboost.mediation.sdk.demo.java;
+
+import android.app.Activity;
+import android.content.Context;
+import android.view.View;
+import android.widget.Button;
+import android.widget.FrameLayout;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.chartboost.chartboostmediationsdk.ChartboostMediationSdk;
+import com.chartboost.chartboostmediationsdk.ad.AdLoadResult;
+import com.chartboost.chartboostmediationsdk.ad.ChartboostMediationAdShowResult;
+import com.chartboost.chartboostmediationsdk.ad.ChartboostMediationBannerAdLoadListener;
+import com.chartboost.chartboostmediationsdk.ad.ChartboostMediationBannerAdLoadRequest;
+import com.chartboost.chartboostmediationsdk.ad.ChartboostMediationBannerAdLoadResult;
+import com.chartboost.chartboostmediationsdk.ad.ChartboostMediationBannerAdView;
+import com.chartboost.chartboostmediationsdk.ad.ChartboostMediationBannerAdViewListener;
+import com.chartboost.chartboostmediationsdk.ad.ChartboostMediationFullscreenAd;
+import com.chartboost.chartboostmediationsdk.ad.ChartboostMediationFullscreenAdListener;
+import com.chartboost.chartboostmediationsdk.ad.ChartboostMediationFullscreenAdLoadListener;
+import com.chartboost.chartboostmediationsdk.ad.ChartboostMediationFullscreenAdLoadRequest;
+import com.chartboost.chartboostmediationsdk.ad.ChartboostMediationFullscreenAdLoadResult;
+import com.chartboost.chartboostmediationsdk.ad.ChartboostMediationFullscreenAdQueue;
+import com.chartboost.chartboostmediationsdk.ad.ChartboostMediationFullscreenAdQueueListener;
+import com.chartboost.chartboostmediationsdk.ad.ChartboostMediationFullscreenAdQueueManager;
+import com.chartboost.chartboostmediationsdk.ad.ChartboostMediationFullscreenAdShowListener;
+import com.chartboost.chartboostmediationsdk.domain.ChartboostMediationAdException;
+import com.chartboost.chartboostmediationsdk.domain.Keywords;
+import com.chartboost.core.ChartboostCore;
+import com.chartboost.core.initialization.ModuleInitializationResult;
+import com.chartboost.core.initialization.ModuleObserver;
+import com.chartboost.core.initialization.SdkConfiguration;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+
+public class AdController {
+
+    private final Map<String, ChartboostMediationFullscreenAd> fullscreenAds = new HashMap<>();
+    private final String TAG = "AdController";
+    private final Map<String, ChartboostMediationFullscreenAdQueue> adQueues = new HashMap<>();
```

Comment:
We might want `ConcurrentHashMap`s. Nothing big since it's a demo but still, wild use cases exist out there.